extension/otto/gohan_core.go

```diff
@@ -109,6 +109,23 @@ func init() {
 		    return this.name.concat("(").concat(this.message).concat(")");
 		  };
 		}
+		function TimeOutException(msg){
+		  BaseException.call(this);
+		  this.message = msg;
+		  this.name = "TimeOutException";
+		  this.code = 408;
```

Comment:
This error happens based on server side overload, so it should return 5xx server error. That should be 503 Service Unavailable. 408 means "The server timed out waiting for the request" so it's different from etcd timeout case.